contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -857,9 +857,14 @@ select
   , cast(imap.index_id as int) as index_id
 from pg_index X 
 inner join index_id_map imap on imap.indexrelid = X.indexrelid
-inner join pg_class I on I.oid = X.indexrelid and (I.relkind = 'i' or I.relkind = 'I') and I.relispartition = false
+inner join pg_class I on I.oid = X.indexrelid
+inner join pg_class ptbl on ptbl.oid = X.indrelid and ptbl.relispartition = false
 inner join pg_namespace nsp on nsp.oid = I.relnamespace
 left join sys.babelfish_namespace_ext ext on (nsp.nspname = ext.nspname and ext.dbid = sys.db_id())
+left join sys.babelfish_partition_depend pd on
```

Comment:
Can we get the performance regression numbers for these changes?

Reply:
I will get the performance numbers for critical views separately.

Reply:
So shall we wait until we get it merged?

Reply:
No, it is not a blocker and can be done later.